src/components/scenes/CoinRankingDetailsScene.tsx

```diff
@@ -488,7 +488,7 @@ const CoinRankingDetailsSceneComponent = (props: Props) => {
             <EdgeText style={styles.title}>{`${currencyName} (${currencyCode})`}</EdgeText>
           </EdgeAnim>
           <SwipeChart assetId={coinRankingData.assetId} />
-          {matchingEdgeAssets.length <= 0 ? null : (
+          {matchingEdgeAssets.length === 0 || hideNonUkCompliantFeat == null ? null : (
             <View style={styles.buttonsContainer}>
```

Comment:
Replace the `View` with an `EdgeAnim` and move the condition above into its `visible` prop